Resources/Views/Authentication/presentation_form.leaf

```diff
@@ -25,12 +25,12 @@
                 </select>
                 <br/>
                 <div class="form-group form-check">
-                <input type="checkbox" class="form-check-input" id="enableSecondSpeaker" data-toggle="toggle">
+                <input type="checkbox" class="form-check-input" id="enableSecondSpeaker" data-toggle="toggle" checked="checked">
```

Comment:
This is later adjusted inside JS, but I found it behaved better if you start with checked